data_portal/urls.py

```diff
@@ -20,6 +20,7 @@
 router.register(r'runs', RunViewSet, basename='runs')
 router.register(r'reports', ReportViewSet, basename='reports')
 router.register(r'presign', PresignedUrlViewSet, basename='presign')
+router.register(r'library', LibraryRunViewSet, basename='library')
```

Comment:
@reisingerf Any comment on `/library` endpoint path? This ... looks okay for me, no strong opinion.

Reply:
Maybe stick with libraryrun to sync with the database table?

Reply:
Yes, for now I'd go with `libraryrun` for consistency and to keep `library` available.

Reply:
Will; after you made changes and ready again for review, pls kindly tick `Re-request review` button, pls. Otherwise, I take that you are still WIP. Thank you. @williamputraintan 

![image](https://user-images.githubusercontent.com/4586509/136473952-71dd13e2-d698-4be4-bab8-4b61ba3a7b29.png)